frontend/src/tests/components/eMIB/ResponseItem.test.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ResponseItem, { RESPONSE_TYPE } from "../../../components/eMIB/ResponseItem";
+import LOCALIZE from "../../../text_resources";
+
+it("renders the right response types (reply, reply all, forward)", () => {
+  //reply
+  const replyWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      cc={"cc"}
```

Comment:
wondering if you should have a test without cc, since it is optional?

Reply:
Otherwise, great UI tests!

Reply:
Good idea!
Added in commit #https://github.com/code-for-canada/project-thundercat/pull/164/commits/7a07fe3afd1ae802edb7b1a8a92fdf03afbce8b6